cluster/charts/universal-crossplane/values.yaml.tmpl

```diff
@@ -13,6 +13,9 @@ nodeSelector: {}
 tolerations: []
```

Comment:
@phisco we still have this file in UXP and have to sync values until we update the release process accordingly right?

Reply:
As we discussed offline, I'll tackle automating/improving this part separately not to block the release.